content/en/docs/setup/install/kubernetes/index.md

```diff
@@ -45,9 +45,6 @@ requirements.
 
 1. Install one of the following variants of the **demo** profile:
 
-{{< tabset cookie-name="profile" >}}
-
-{{< tab name="permissive mutual TLS" cookie-value="permissive" >}}
 When using the [permissive mutual TLS mode](/docs/concepts/security/#permissive-mode), all services accept both plaintext and
```

Comment:
Take out all this text, since the user doesn't need to make a choice anymore. Just leave the command.

Reply:
Done. Thanks.